src/containers/Chat/ChatMessages/ChatInput/ChatInput.tsx

```diff
@@ -195,14 +202,17 @@ export const ChatInput: React.SFC<ChatInputProps> = (props) => {
     setAttachmentType('');
   };
 
-  const handleSelectText = (obj: any) => {
+  const handleSelectText = (obj: any, isInteractiveMsg: boolean = false) => {
     resetVariable();
 
-    // set selected template
-    setSelectedTemplate(obj);
-
-    // Conversion from HTML text to EditorState
-    setEditorState(EditorState.createWithContent(WhatsAppToDraftEditor(obj.body)));
+    if (isInteractiveMsg) {
+      submitMessage(obj.interactiveContent, obj.type);
+    } else {
```

Comment:
```suggestion
    if (isInteractiveMsg) {
      submitMessage(obj.interactiveContent, obj.type);
      return;
    } else {
```

Let's add a return in if since we don't need to run the further lines if it goes in if